tensilelite/rocisa/rocisa/src/instruction/mfma.cpp

```diff
@@ -24,15 +24,81 @@
 
 #include <nanobind/nanobind.h>
 #include <nanobind/stl/optional.h>
+#include <nanobind/stl/pair.h>
 #include <nanobind/stl/shared_ptr.h>
 #include <nanobind/stl/string.h>
 #include <nanobind/stl/variant.h>
 #include <nanobind/stl/vector.h>
 
 namespace nb = nanobind;
 
+namespace rocisa
+{
+    DataType instTypeToDataType(InstType instType)
+    {
+        switch(instType)
+        {
+        case InstType::INST_F16:
+            return DataType::Half;
+        case InstType::INST_F32:
+            return DataType::Float;
+        case InstType::INST_F64:
+            return DataType::Double;
+        case InstType::INST_BF16:
+            return DataType::BFloat16;
+        case InstType::INST_I8:
+            return DataType::Int8;
+        case InstType::INST_U8:
+            return DataType::Int8;
+        case InstType::INST_I32:
+            return DataType::Int32;
+        case InstType::INST_XF32:
+            return DataType::XFloat32;
+        case InstType::INST_F8:
+            return DataType::Float8;
+        case InstType::INST_BF8:
+            return DataType::BFloat8;
+        case InstType::INST_F8_BF8:
+            return DataType::Float8BFloat8;
+        case InstType::INST_BF8_F8:
+            return DataType::BFloat8Float8;
+        default:
+            throw std::runtime_error("Unknown instruction type");
+        }
+    }
+
+    bool is8bitFloat(DataType value)
+    {
+        switch(value)
+        {
+        case DataType::Float8:
+        case DataType::BFloat8:
+        case DataType::Float8BFloat8:
+        case DataType::BFloat8Float8:
+        case DataType::Float8_fnuz:
+        case DataType::BFloat8_fnuz:
+        case DataType::Float8BFloat8_fnuz:
+        case DataType::BFloat8Float8_fnuz:
+            return true;
+        default:
+            return false;
+        }
+    }
+} // namespace rocisa
+
 void mfma_inst(nb::module_ m_mfma)
 {
+    m_mfma
+        .def("getMFMAIssueLatency",
+             &rocisa::getMFMAIssueLatency<false>,
+             nb::arg("dataType"),
+             nb::arg("miM"),
+             nb::arg("miB"))
```

Comment:
you get miM and miB from variant[] ? Should we still need these args?

Reply:
This is a function used by the mfma classes.